### ISSUE TITLE
Quest: Hidden Enemies

### ISSUE THREAD

**Issue by underlord-oberon** [plantain-gnocchis/wow-voiceover-tbc-wotlk#82]
Add-on version: AI_VoiceOver_v1.4.5
Data version: AI_VoiceOverData_VanillaExtra_v1.4

- Orgrimmar / Cleft of Shadows / NPC: Neeru Fireblade / Quest: Hidden Enemies
  Link: https://www.wowhead.com/wotlk/quest=5727/hidden-enemies

  Description: playback ok.

  Progress: no playback.

  Completion: playback ok.

- Talking with Neeru:

  "You may speak frankly..." text: playback ok.

  "It is good to see the Burning Blade..." text: playback ok.

  "So the Searing Blade is expendable?" text: playback the wrong voice-over. The second one of this dialog sequence.

  "If there is anything you would have of me..." text: playback the wrong voice-over. The second one of this dialog sequence.

**Comment by carevass** [comment]
This should be fixed with new audio in AI_VoiceOverData_VanillaExtra_v1.5.
